torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -220,7 +221,8 @@ class TensorExprFuser {
     removeProfileNodesAndSpecializeTypes(graph_->block());
     GRAPH_DUMP(
         "After removing profiling nodes and specializing types: ", graph_);
-    createFusionGroups(graph_->block());
+    RemoveTensorMutation(graph_);
```

Comment:
Perhaps this should be renamed - it's really `TryRemoveTensorMutation` in that it only removes tensor mutation if it can prove this is a safe transformation. It doesn't guarantee that the graph will not contain mutable operations.